joomla/joomla-cms
# [com_content] Complete search tooltip

#### Summary of Changes

Simple PR to improves the search tooltip and element label (accessibility) in com_content.
Also add the ability to search for author in `featured` view to have the same behaviour as the `articles` view.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Content -> Articles, check the tooltip in the search field.
3. Go to Content -> Featured, check the tooltip in the search field.
4. Go to frontend edit, click "+Article" in TinyMce and check the tooltip in the search field of the popup.
